### PR TITLE
Season changes automatically

### PR DESCRIPTION
**Pull request by Melody-Ruth**
The textures on the map gradually change to match the seasons (using Ben's system for setting blended textures). Pure spring textures should occur in the middle of the spring season, pure summer textures in the middle of the summer season, etc.
Also includes Joanne's updated textures for the tall trees.

### CODE REVIEW

**Review by Alanine42 — APPROVED** [review]
looks good.

interesting 
image

File: common/GameConstants.h [hunk 1]
```diff
@@ -90,3 +90,5 @@ enum MovementEntityStateIndex {
 
 // Lerping
 #define LERP_DURATION_TICKS 4.0f
+
+#define SEASON_LENGTH 500
```
Comment (benjxia) [inline common/GameConstants.h:94]: What units are these in again

Comment (Alanine42) [reply common/GameConstants.h:94]: server ticks

**Review by benjxia — CHANGES_REQUESTED** [review]
I don't think this branch is up to date with main yet. I get the blue screen issue when using character abilities.

**Comment by Alanine42** [comment]
yep, we gotta merge main

**Comment by Melody-Ruth** [comment]
> yep, we gotta merge main
> 
> can @Melody-Ruth merge this? If i merge then my approval won't count
> 
> thanks

oops I forgot. Doing it now

**Comment by Melody-Ruth** [comment]
Ok it automerged since it's completely separate changes. Should be ready for review

**Review by benjxia — APPROVED** [review]

**Pull request merged**